plugin/dokuvimki.py

```diff
@@ -9,6 +9,11 @@
 import time
 import subprocess
 
+from PIL import ImageGrab
```

Comment:
In this case I think moving the import *into* the `paste_image` function might be prudent: with a top-level import, `PIL` becomes a required dependency even if one never uses pasting from the clipboard. wdyt?

Reply:
I've used the same way like xmlrpc check and included a check inside the pasteimage function including error message on missing module.